src/styleProps/textProps.ts

```diff
@@ -7,37 +7,39 @@ const textTransformMap = {
 	none: 'none',
 };
 
+type TextTransformKey = keyof typeof textTransformMap;
+
 export const textPropsDef = [
 	{
 		styleProp: 'bold',
 		properties: ['fontWeight'],
-		getValue: bold => (bold ? 'bold' : 'normal'),
+		getValue: (bold: unknown) => (bold ? 'bold' : 'normal'),
```

Comment:
I think this can be `boolean`.

Reply:
This is the same as a couple other places where the problem lies deeper. I think IStylesPropsConfig is where we are running into issues and we'd need come back and properly define the interface and then update the code that uses it to adhear to it. `unknown` here gives us an easy out to make progress before we do that scope of work.